doc/user/layouts/partials/head.html

```diff
@@ -93,57 +93,19 @@
 </script>
 
 {{if hugo.IsProduction}} {{/* Google Analytics */}}
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-HYXTGZN2KT"></script>
 <script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'G-HYXTGZN2KT');
-</script>
-
-{{/* Leadfeeder */}}
-<script>
-  (function () {
-    window.ldfdr = window.ldfdr || {};
-    (function (d, s, ss, fs) {
-      fs = d.getElementsByTagName(s)[0];
-      function ce(src) {
-        var cs = d.createElement(s);
-        cs.src = src;
-        setTimeout(function () {
-          fs.parentNode.insertBefore(cs, fs);
-        }, 1);
-      }
-      ce(ss);
-    })(
-      document,
-      "script",
-      "https://sc.lfeeder.com/lftracker_v1_bElvO73R0rE8ZMqj.js"
-    );
-  })();
-</script>
-{{end}} {{/* Segment */}}
-<script>
-  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on","addSourceMiddleware","addIntegrationMiddleware","setAnonymousId","addDestinationMiddleware"];analytics.factory=function(e){return function(){var t=Array.prototype.slice.call(arguments);t.unshift(e);analytics.push(t);return analytics}};for(var e=0;e<analytics.methods.length;e++){var key=analytics.methods[e];analytics[key]=analytics.factory(key)}analytics.load=function(key,e){var t=document.createElement("script");t.type="text/javascript";t.async=!0;t.src="https://cdn.segment.materialize.com/analytics.js/v1/" + key + "/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(t,n);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.15.3";
-  {{ if hugo.IsProduction }}
-    {{/* Prod/Stage */}}
-    var SEGMENT_ID = 'NCe6YQCHM9g04X9yEBUFtoWOuqZU8J1m';
-    if(window.location.hostname.includes('preview')) {
-      SEGMENT_ID = 'fuo8pqI0wvoojiShBQXSZopWLZggS8ny';
-    }
-    analytics._writeKey=SEGMENT_ID;
-    analytics._cdn="https://cdn.segment.materialize.com";
-    {{/* Use cookies for storage by default, since they cross subdomains */}}
-    analytics.load(SEGMENT_ID, { storage: { stores: ["cookie", "localStorage", "memory"] } });
-  {{ else }}
-    {{/* Dev */}}
-    analytics._writeKey="dGeQYRjmGVsqDI0KIARrAhTvk1BdJJhk";
-    analytics.load("dGeQYRjmGVsqDI0KIARrAhTvk1BdJJhk", { storage: { stores: ["cookie", "localStorage", "memory"] } });
-  {{ end }}
-  analytics.page();
-  }}();
+  (function (w, d, s, l, i) {
```

Comment:
I notice that the existing code has `{{ if hugo.IsProduction }}`. I wonder if we also want that here?

Reply:
Sorry I'm unable to understand what you meant here that is still there wrapping the script tag.